### ISSUE TITLE
Don't show snippets for user tags with spaces in name

### ISSUE THREAD

**Issue by datho7561** [redhat-developer/quarkus-ls#787]
If you have a user tag with spaces in it's name, like:

`src/main/resources/templates/tags/my tag.html`

Then a snippet for `my tag` appears. When you apply the snippet, qute-ls reports that the generated code is an error.

It's not clear from the Qute documentation what should be done in this case, however accepting the snippet and trying to render the template results in an error. Also, trying variations on the name like `my-tag`, `mytag`, `myTag` still results in the template failing to render.

**Comment by angelozerr** [comment]
@mkouba @ia3andy what is the rule when HTML file name contains space and it is hosted in tags folder? Shous we ignore this user tag in completion?

**Comment by mkouba** [comment]
> @mkouba @ia3andy what is the rule when HTML file name contains space and it is hosted in tags folder? Should we ignore this user tag in completion?

Indeed, this is undefined. And I think that we should ignore these template files completely, not only in qute-ls but also in the `quarkus-qute` extension.

The only question is how strict to be. A reasonable approach is IMO to accept a sequence of non-whitespace characters (we use the same rule for [identifiers](https://quarkus.io/guides/qute-reference#identifiers)).
